getting_started/setup/setup_linux_machine.rst

```diff
@@ -0,0 +1,142 @@
+.. _setup-linux-machine:
+
+Set up a Linux dev and test environment
+#######################################
+
+.. contents::
+   :local:
+   :depth: 3
+
+Introduction
+************
+
+
+These instructions will guide you through the process of building the SOF branch of the Linux kernel from scratch and installing it on your machine. The kernel will be installed in addition to your default kernel, so if something goes wrong you can always revert to the default kernel.
+
+Prerequisites
+*************
+
+1. Device requirements
+-----------------------
+
+PC with running Fedora (version 35 or later recommended).
+
+TODO: What other system requirements are there?
```

Comment:
Ubuntu 21.04. we'll need instructions for those systems

Reply:
Alright, I'll install and test that now

Reply:
Added instructions for Ubuntu and verified them 👍